kolibri_explore_plugin/assets/src/views/ContentModal.vue

```diff
@@ -116,6 +116,12 @@
     border-bottom-left-radius: $modal-content-border-radius;
   }
 
+  ::v-deep .modal-content .paragraph {
+    // The color used in the perseus plugin:
+    // https://github.com/learningequality/ka-lite/blob/571918ea668013dcf022286ea85eff1c5333fb8b/kalite/distributed/static/js/distributed/perseus/build/perseus-5.css#L60
+    color: #21242c;
```

Comment:
Why use this color instead of $dark from our theme?

Reply:
I used this color to match the color of the questions, as I put in the comment, this is the color used by the plugin that renders the the exercise. It's possible to use the `$dark` value, but in that case the color of the test description will not match the answers color.

Reply:
I see, please communicate with Joana in that case. Maybe she wants the 2 description and answer to use our theme color. See her spec https://phabricator.endlessm.com/T32300

Reply:
I've used the `$gray-700` for all paragraphs in the modal, so it's the same for question and answers.